src/front-end/providers/toast-provider.tsx

```diff
@@ -1,13 +1,13 @@
 import React, { createContext, type FC, type ReactNode, useCallback, useContext, useState } from 'react';
 import type { ProviderProps } from '@/front-end/types/provider-types.ts';
 import { Toast } from 'radix-ui';
+import { getKey } from '@/front-end/lib/utils.ts';
 
-// TODO: move this guy?
 const typeColorMap = new Map<string, string>([
-  ['info', 'bg-blue-300'],
-  ['success', 'bg-green-300'],
-  ['warning', 'bg-amber-300'],
-  ['error', 'bg-red-300'],
+  ['info', 'bg-blue-400'],
+  ['success', 'bg-green-400'],
+  ['warning', 'bg-amber-400'],
+  ['error', 'bg-red-400'],
 ]);
```

Comment:
This would probably be better as an enum?

Reply:
Can you explain? since its just a string-string map i feel like an enum would be better so you have better typesafety and you can do it on compiletime instead of at runtime